src/SFML/Window/iOS/EaglContext.hpp

```diff
@@ -159,12 +159,12 @@ class EaglContext : public GlContext
     ////////////////////////////////////////////////////////////
     // Member data
     ////////////////////////////////////////////////////////////
-    EAGLContext* m_context;      ///< The internal context
-    GLuint       m_framebuffer;  ///< Frame buffer associated to the context
-    GLuint       m_colorbuffer;  ///< Color render buffer
-    GLuint       m_depthbuffer;  ///< Depth render buffer
-    bool         m_vsyncEnabled; ///< Vertical sync activation flag
-    Clock        m_clock;        ///< Measures the elapsed time for the fake v-sync implementation
+    EAGLContext* m_context;             ///< The internal context
```

Comment:
Any reason this member is not initialized here?

Reply:
You can't put that initial value `nil` in this C++ header. I tried and it failed to build. Maybe we could use `nullptr` as the initial value but idk much about Objective-C++.

Reply:
https://github.com/ChrisThrasher/SFML/actions/runs/3505531814

I tried this out and CI passes but I'm not confident enough about the differences between `nil` and `nullptr` to make this change.